hebiros_gazebo_plugin/include/hebiros_gazebo_joint.h

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
I love seeing classes like this -- smaller, concise classes really help separate out the structure and clarify the logic